master.adoc

```diff
@@ -0,0 +1,4 @@
+= People's Liberation Training Committee Style Guide
```

Comment:
When rendered, jamming PLTCOM and style guide doesn't look good. Maybe try this?
```suggestion
= People's Liberation Training Committee: Style Guide
```